env.js

```diff
@@ -138,7 +139,9 @@ Promise.resolve()
 
     return writeJSON(adapterPackageJsonPath, adapterJson, true).then(() => Promise.all([
       // npm install the peer deps at the root
-      run('npm', 'i', '--no-save', ...installs),
+      Promise.all(
+        rmrfs.map(s => primraf(s))
+      ).then(() => run('npm', 'i', '--no-save', ...installs)),
```

Comment:
The previous `npm i` in this script readds the `node_modules/npm` link... so it has to be removed before this `npm i --no-save` otherwise the same problem just happens again.